drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/commands/actualcommands/AbstractSelectedColumnCommand.java

```diff
@@ -48,10 +50,17 @@
  */
 public abstract class AbstractSelectedColumnCommand extends AbstractScenarioSimulationCommand {
 
+    protected FactMappingValueType factMappingValueType = FactMappingValueType.RAW;
```

Comment:
@yesamer 
Pls remove that - there is no real cause-effect relationship for a command to be a "SelectedColumn" command and the kind of data modified by the command.